azure-pipelines.yml

```diff
@@ -0,0 +1,121 @@
+trigger:
+- v2
+
+jobs:
+
+  - job: V1Test
```

Comment:
I suggest we name this `Functions 1.0 Tests` or something similar. `V1` by itself can be a bit ambiguous.